src/components/feedback/feedback.js

```diff
@@ -18,14 +19,13 @@ export default function FeedbackComponent(props) {
   return (
     <>
       <section
-        className={`nsw-section nsw-section--box nsw-bg--off-white nsw-position-absolute nsw-position-left-0 ${
-          isSubmitted ? 'nsw-display-none' : ''
+        className={`nsw-section nsw-section--box nsw-bg--off-white feedback-container ${
```

Comment:
this is changes for the feedback, but this is performanc branch? i dont mind to combine, i just point this out whether you commit correct code?